components/brave_ads/core/internal/ads/serving/choose/sample_ads_unittest.cc

```diff
@@ -14,6 +16,28 @@
 
 namespace brave_ads {
 
+TEST(BatAdsSampleAdsTest, CalculateNormalizingConstantBaseInteger) {
+  // Arrange
+  const std::vector<int> vector = {1, 2, 3, 4, 5};
+
+  // Act
+  const int normalizing_constant = CalculateNormalizingConstantBase(vector);
+
+  // Assert
+  EXPECT_EQ(15, normalizing_constant);
+}
+
+TEST(BatAdsSampleAdsTest, CalculateNormalizingConstantBaseDouble) {
+  // Arrange
+  const std::vector<double> vector = {1.3, 2.7, 3.1, 4.8, 5.2};
+
+  // Act
+  const double normalizing_constant = CalculateNormalizingConstantBase(vector);
```

Comment:
`CalculateNormalizingConstant` and rename the current implementation to `CalculateNormalizingConstantFromPredictors`